INSTALL.md

```diff
@@ -1259,7 +1259,19 @@ the same.
 
 #### OpenVMS
 
-    $ @config [options...]
+    $ perl Configure [options...]
+    
+    On OpenVMS explicit versions for 32/64-bit pointers can be build. To do so you 
+    should use one of the following options:
+    on AXP:
+         vms-alpha  (default option)
+         vms-alpha-p32
+         vms-alpha-p64
+    on IA64:
+         vms-ia64 (default option)
+         vms-ia64-p32
+         vms-ia64-p64
+    on X86: not yet implemented)
```

Comment:
I would place this a little further down, in the "Manual Configuration" section seen below, or in NOTES-VMS.md.

Reply:
OK to me